docker-compose.yml

```diff
@@ -40,7 +42,7 @@ services:
       - '2222:22'
     depends_on:
       - postgres
-    command: /bin/bash --verbose -c "cd /go/src/code.gitea.io/gitea/ && make backend && cp gitea /app/gitea/ && /usr/bin/entrypoint"
+#    command: /bin/bash --verbose -c "cd /go/src/code.gitea.io/gitea/ && make backend && cp gitea /app/gitea/ && /usr/bin/entrypoint"
 
   postgres:
     image: postgres:9.6
```

Comment:
Please remove these docker-compose changes from this branch, we'll figure this out separately.